client/src/redux/settings/settings-sagas.js

```diff
@@ -113,6 +113,7 @@ function* updateMySoundSaga({ payload: update }) {
 function* updateMyThemeSaga({ payload: update }) {
   try {
     const { data } = yield call(putUpdateMyTheme, update);
+    localStorage.setItem('theme', update.theme);
```

Comment:
There is no need to access the theme, pass it through components, invert it, and pass it to updateMyTheme.
updateMyTheme can access the theme directly, invert it and reset it.
Calling action toggleTheme would be the most appropriate.